src/Symfony/Component/VarDumper/Tests/Caster/RedisCasterTest.php

```diff
@@ -17,6 +17,7 @@
 /**
  * @author Nicolas Grekas <p@tchwork.com>
  * @requires extension redis
+ * @requires extension relay
```

Comment:
better figure out away to run only the redis tests when relay isn't available, and vice versa